l10n_fr_hr/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 GRAO
+#   Quentin DUPONT <https://twitter.com/pondupont>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "French Localization for HR",
+    "summary": "French Localization for HR",
+    "version": "12.0.1.0.0",
+    "category": "French Localization",
+    "author": "GRAP,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["hr_cae", "l10n_fr_department"],
```

Comment:
Regarding this module, I think that :
- the name should be l10n_fr_department_hr (as it will come in to l10n-france, all the module should be prefixed by l0n_fr).
- should not depend on ``hr_cae``, but depend only on ``hr``. 
1) I don't see what it is required in ``hr_cae``
2) this module will not be accepted, in OCA, if depends on non OCA modules.
- auto_install should be False, because maybe some user doens't want to see this module installed, if they don't requires there fields. ``auto_install = True`` is generally only set if module A and module B doesn't works and need a fix if installed together. (or if there is no reason to not install the glue module, if A & B are installed).

Reply:
Ok I missunderstand glue modules.
Thanks for explanation :ok_hand:

Reply:
Depends on `hr`, hr.employee is a model in `hr`

Reply:
ok.

Reply:
So `l10n_fr_department_hr_employee` it can't work just depending on `hr` since we add the `country_of_birth` field in the hr_cae view..

What's the best way to do this ? 

Add in oca hr/hr.employee view
Then change it in hr_cae view making `hr_cae` depending on `l10n_fr_department_hr_employee` ?? :confused:

Reply:
Mmh, if you're factorizing, it'd be a pity to add french dependencies into hr_cae. Maybe `hr_cae_fr`?